### PR TITLE
Reader | Highlight search results

### PR DESCRIPTION
**Pull request by dannysellers**
Connects #3658 

Highlights all instances of search term on value change

### CODE REVIEW

File: client/app/reader/PdfPage.jsx [hunk 1]
```diff
@@ -180,6 +191,10 @@ export class PdfPage extends React.PureComponent {
       this.clearPage();
     }
     this.previousShouldDraw = shouldDraw;
+
+    if (this.markInstance) {
+      this.debouncedMarkText(this.props.searchText);
```
Comment (mdbenjam) [inline client/app/reader/PdfPage.jsx]: Why did you add the debounce? Was it too slow otherwise. It feels really laggy right now.

Comment (dannysellers) [reply client/app/reader/PdfPage.jsx]: Without debouncing, it felt like I was getting some lag when searching, yeah (though that may have been with small search terms in a large document).

Unless a user searches for the same thing multiple times in the same session (taking advantage of selector memoization), I figured it's more consistent to wait a split second after the user finishes typing, rather than to run a search for every keystroke. I haven't actually timed it, though, and don't feel particularly strongly about it either way.

Also, I'm not very familiar with the `redux-search` library, but if we don't debounce, I'd guess a user might end up queueing multiple searches if they type quickly enough, which I could see being laggy

Comment (mdbenjam) [reply client/app/reader/PdfPage.jsx]: I would see what it does on a 6 page document. With the windowing changes coming up that should be the max: https://github.com/department-of-veterans-affairs/caseflow/issues/3762. I don't plan on turning this feature on until that's ready.

Comment (dannysellers) [reply client/app/reader/PdfPage.jsx]: Oh right, I forgot about your windowing changes. That should obviate the need for debouncing, definitely. Pushing a fix..
